exercises/macros/tests/macros.rs

```diff
@@ -67,6 +67,31 @@ fn test_nested() {
     );
 }
 
+#[test]
+#[ignore]
+fn test_type_override() {
+    // The macro should always use std::collections::HashMap and ignore crate::std::collections::HashMap
+    mod std {
+        pub mod collections {
+            pub struct HashMap();
```

Comment:
```suggestion
            pub struct HashMap;
```

I believe that both declarations properly compile down to a ZST, but this form makes it more obvious.